curious/__init__.py

```diff
@@ -68,15 +81,29 @@ def __add_model_by_class(self, cls, short_name=None):
       self.__short_names[manager.short_name].append(manager)
 
   def register(self, model, short_name=None):
-    if type(model) == types.ModuleType:
+    if isinstance(model, types.ModuleType):
```

Comment:
This is not an innocuous change. isinstance will check subclasses whereas the existing check will not.

Reply:
yes, but this is `types.ModuleType` we're talking about here: if something subclasses the _module_ type, it should behave as such, so we're making the behavior more reasonable here, even though the point is mostly moot, as no one subclass `module` anyway.  since `isinstance` is the more idiomatic way Python behaves, I don't see a need to change it back